Gruntfile.js

```diff
@@ -391,6 +434,22 @@ module.exports = function (grunt) {
                     "echo '--- Config scripts finished. ---\n'"
                 ].join(";")
             },
+            generateNodeIndex: {
+                command: [
+                    "echo '\n--- Regenerating node index ---'",
+                    // Why copy and wipe OperationConfig?
+                    // OperationConfig.json needs to be empty for build to avoid circular dependency on DetectFileType.
+                    // We copy it to node dir so that we can use it as a search corpus in chef.help.
+                    "echo 'Copying OperationConfig.json and wiping original'",
```

Comment:
Don't like this. It seems that dynamic `import()` statements are now possible in Node 9.7.x. Would this fix the problem?